tests/nexus/test_build_dxtbx_models.py

```diff
@@ -177,8 +177,8 @@ def test_get_dxtbx_scan(nxmx_example):
     scan = dxtbx.nexus.get_dxtbx_scan(sample, instrument.detectors[0])
     assert scan.get_num_images() == 10
     assert scan.get_image_range() == (1, 10)
-    assert scan.get_oscillation() == (0.0, 0.1)
-    assert scan.get_oscillation_range() == (0.0, 1.0)
+    np.testing.assert_allclose(scan.get_oscillation(), (0.0, 0.1))
+    np.testing.assert_allclose(scan.get_oscillation_range(), (0.0, 1.0))
```

Comment:
Is there a particular reason you didn't use `pytest.approx` here?
```suggestion
    assert pytest.approx(scan.get_oscillation()) == (0.0, 0.1)
    assert pytest.approx(scan.get_oscillation_range()) == (0.0, 1.0)
```

Reply:
None, I just forgot it exists!